src/vma/util/data_updater.cpp

```diff
@@ -79,3 +79,22 @@ bool header_tos_updater::update_field(dst_entry &dst)
 	return true;
 }
 
+ring_alloc_logic_updater::ring_alloc_logic_updater(int fd, lock_base & socket_lock,
+						   resource_allocation_key & ring_alloc_logic,
+						   socket_stats_t* socket_stats)
+	: data_updater()
+	, m_fd(fd)
+	, m_socket_lock(socket_lock)
+	, m_key(ring_alloc_logic)
+	, m_sock_stats(socket_stats)
+{
+
+}
+
+bool ring_alloc_logic_updater::update_field(dst_entry &dst)
```

Comment:
Do we want to return true for each and every call ?

Reply:
So does the other updaters do... in any case, nobody is looking at the return value (See sockinfo_udp/tcp.cpp at sockinfo_udp::update_header_field